app/models/manageiq/providers/native_crud.rb

```diff
@@ -0,0 +1,129 @@
+class ManageIQ::Providers::NativeCrud < Job
```

Comment:
+1 for not `Crud`

I guess it's for any operation thats to be scheduled (async) and which needs a refresh after it.